### ISSUE TITLE
Tiago's comments on evaluating the models

### ISSUE THREAD

**Issue by kamilest** [kamilest/brain-age-gnn#71]
Saving here so it's not lost in messages.

> Btw, I just thought about something I didn't tell in our last zoom meeting. At this moment you said that in order to get a single model, you can average all the weights/biases of your models across the different folds (so you have a final metric in the test set). I have to say that I'm not sure how that will work out, because we don't know how weights might depend on other weights, thus you might get a completely different model 😞 I think averaging weights is used in NNs, but if you find problems, I suggest something to you, which is what happens when you set refit=True in GridSearchCV (the default behaviour actually): after you select the best set of parameters, you fit a model (with those hyperparameters selected previously) on the entire training set. In NNs this is a problem I guess because of the number of epochs, but keep this in mind in case you run out in problems
